modyn/supervisor/internal/pipeline_executor/evaluation_executor.py

```diff
@@ -156,7 +156,9 @@ def run_pipeline_evaluations(
             )
             eval_requests += handler_eval_requests
 
-        num_workers = self.config.supervisor.after_training_evaluation_workers if self.config.supervisor else 1
+        if len(eval_requests) == 0:
+            return SupervisorLogs()
+        num_workers = self.pipeline.evaluation.after_training_evaluation_workers
```

Comment:
here we don't need to assert on self.pipeline.evaluation, as it is asserted in the beginning of this func.